test.sh

```diff
@@ -500,6 +500,11 @@ function test_docs {
   diff docs/reference/includes $kb_orig/test/docs/expected/includes
   diff docs/reference/manifest.json $kb_orig/test/docs/expected/manifest.json
   diff docs/reference/config.yaml $kb_orig/test/docs/expected/config.yaml
+
+  header_text "testing doc annotations"
+  sed -i -e "s|// +kubebuilder:categories=foo,bar|// +kubebuilder:categories=foo,bar\n// +kubebuilder:doc:note=test notes message annotations\n// +kubebuilder:doc:warning=test warnings message annotations|" pkg/apis/insect/v1beta1/bee_types.go
```

Comment:
Would you add a test that has the annotations on 2 different resources (e.g. add an annotation to Ant as well).

Reply:
you can do it with `i` or `a` command in `sed` with something like this:
```
sed -i '/type [a-zA-Z]* struct/ i NEWLINE' FILE_TO_MODIFY
```